Documentation/guides/messages/xa1004.md

```diff
@@ -0,0 +1,39 @@
+# Compiler Warning XA1004
+
+When [Layout Bindings and Code-Behind][code-behind] are enabled, this warning
+will be emitted for every layout element that has the `//*/@android:id`
+attribute set and uses a fully-qualified name for the element type.
+
+For example, the warning *will* be emitted for both of the following elements:
+
+```xml
+<android.widget.TextView
+    android:id="@+id/text1" />
+<Android.Widget.TextView
+    android:id="@+id/text2" />
+```
+
+But it will *not* be emitted for elements that use just an unqualified class
+name like:
+
+```xml
+<TextView
+    android:id="@+id/text1" />
+```
+
+The "naive type name fixup" tries to ensure that any fully-qualified type name
```

Comment:
should `naive` be `native` ?

Reply:
Good sanity-check :-) In this case as I understand it, it is indeed intended to be "naive" because the fixup approach is fairly simplistic. Context:
https://github.com/xamarin/xamarin-android/blob/14d91c314f9c9f2ba7b1d69556b6b2044afb6ab5/src/Xamarin.Android.Build.Tasks/Tasks/CalculateLayoutCodeBehind.cs#L320-L331